modules/reitit-ring/src/reitit/ring.cljc

```diff
@@ -65,6 +65,8 @@
       ([request respond _]
        (respond (handle request))))))
 
+(def ring-endpoint? (r/mk-intermediate-endpoint-predicate #{:handler}))
```

Comment:
would `:name` be an endpoint? routes don't have to have a handler...

Reply:
Changed it so that `:name` signifies endpoint and is consumed by the intermediate path, i.e., not passed on to child routes.